kyo-os-lib/shared/src/main/scala/kyo/os.scala

```diff
@@ -0,0 +1,213 @@
+package kyo
+
+import java.io.*
+import java.lang.Process as JProcess
+import java.lang.ProcessBuilder.Redirect
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Path
+import java.util.concurrent.TimeUnit
+import kyo.*
+import scala.jdk.CollectionConverters.*
+
+sealed trait Command:
```

Comment:
We use `Command` for values used in effect suspensions, what do you think about renaming this to `ProcessCommand` to avoid cofusion?